docs/user/reference/general/initializing.md

```diff
@@ -86,7 +86,7 @@ To learn about SDK specific configuration options available, refer to the [Refer
 >
 > Always pass the user preference, e.g. `Glean.initialize(uploadEnabled=userSettings.telemetryEnabled)` or the equivalent for your application.
 >
-> Calling `Glean.setUploadEnabled(false)` at a later point will trigger [`deletion-request` pings](../../user/pings/deletion-request.md) and regenerate client IDs.
+> Calling `Glean.setCollectionEnabled(false)` at a later point will trigger [`deletion-request` pings](../../user/pings/deletion-request.md) and regenerate client IDs.
```

Comment:
I guess _technically_ we don't regenerate the client_id until `setCollectionEnabled(true)` is called. Might be worthwhile to discuss what we do with the data here also.